Spigot-Server-Patches/0215-PlayerAttemptPickupItemEvent.patch

```diff
@@ -16,19 +16,30 @@ index fed1f4ab2..a580a5866 100644
  import com.destroystokyo.paper.HopperPusher; // Paper
  
  // Paper start - implement HopperPusher
-@@ -329,6 +330,12 @@ public class EntityItem extends Entity implements HopperPusher {
-             Item item = itemstack.getItem();
-             int i = itemstack.getCount();
- 
+@@ -332,7 +333,22 @@ public class EntityItem extends Entity implements HopperPusher {
+             // CraftBukkit start - fire PlayerPickupItemEvent
+             int canHold = entityhuman.inventory.canHold(itemstack);
+             int remaining = i - canHold;
+-            boolean flyAtPlayer = false; // Paper
++
```

Comment:
Empty space isn't mine. It's just the diff is weird (it removes empty space but adds it back?)